recipe-app/View/CustomCollectionViewCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CustomCollectionViewCell.swift
+//  recipe-app
+//
+//  Created by Arthur Hermann on 29/11/2024.
+//
+
+import UIKit
+
+final class CustomCollectionViewCell: UICollectionViewCell {
+    
+    static let identifier = "CustomCollectionViewCell"
+    
+    let myImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.image = UIImage(systemName: "image")
+        imageView.tintColor = .white
+        imageView.clipsToBounds = true
+        return imageView
+    }()
+    
+    func setUp() {
+        self.addSubview(myImageView)
+        myImageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
```

Comment:
We can create a separate method for configuring constraint so it's clear what each method does. 
```suggestion
        private lazy var myImageView: UIImageView = {
        let imageView = UIImageView()
        imageView.contentMode = .scaleAspectFill
        imageView.image = UIImage(systemName: "image")
        imageView.tintColor = .white
        imageView.clipsToBounds = true
        imageView.translatesAutoresizingMaskIntoConstraints = false
        return imageView
    }()
    
    private func setUp() {
        self.addSubview(myImageView)
        configureConstraints()
    }
    
    private func configureConstraints() {
        NSLayoutConstraint.activate([
            myImageView.topAnchor.constraint(equalTo: self.topAnchor),
            myImageView.bottomAnchor.constraint(equalTo: self.bottomAnchor),
            myImageView.trailingAnchor.constraint(equalTo: self.trailingAnchor),
            myImageView.leadingAnchor.constraint(equalTo: self.leadingAnchor)
        ])
    }
```